### DISCUSSION TITLE
Can you add this into lazer?

### DISCUSSION THREAD

**Discussion by not-ollyo** [ppy/osu#24201] [Ideas] [answered]
I suggest that add these stars that popup when reach to some points of the track, just amazing to see it in lazer when idle

https://github.com/ppy/osu/assets/139444720/b90ecfef-e2c6-4a44-bc7e-5cbc0f54a12b

**Comment by peppy** [comment]
Yep, along with menu snow, this needs to return.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/24208

**Reply by not-ollyo** [reply]
you're the best developer of all time, keep it up<3
